ccbuilder/__init__.py

```diff
@@ -176,9 +182,13 @@ def ccbuilder_cache_parser() -> ArgumentParser:
     parser = ArgumentParser(add_help=False)
     parser.add_argument(
         "action",
-        choices=("clean", "stats"),
+        choices=("clean", "stats", "scan"),
```

Comment:
Now that there's the `cstore`, having and also exposing to the CLI a `add` (exists) and a `remove` operation might be useful, for example when pruning the cache.

Reply:
What would these exactly do? 

- `add` adds a manually built compilers? 
- `remove` removes it from the DB and also deletes it from disk?